### ISSUE TITLE
import/extensions: does not complain about .vue file

### ISSUE THREAD

**Issue by 3nuc** [benmosher/eslint-plugin-import#1533]
Hey, I was trying to set up `import/extensions` rule. I wanted to make it so that only .vue files require an extension in the import.

This is similar to #583 , just for .vue

Test case:

```js
{
  'import/extensions': ['error', 'never', { vue: 'always', js: 'always'}]
}
```

```js
import HelloWorld from '../components/HelloWorld' //vue file, no error
import a from '../components/stuff' //js file, errors correctly
```

[My package.json and .eslintrc.js when testing](https://gist.github.com/3nuc/3766afa7bc5a8511bc3c5bb05615caf9)

**Comment by 3nuc** [comment]
Same behavior for

```
"import/extensions": ["error", "ignorePackages", { "vue": "always", js: "always" }],
```

and for the two imports from the above questions.

**Comment by ljharb** [comment]
See the test case that closes that similar issue: https://github.com/benmosher/eslint-plugin-import/pull/1010/files

you need to also configure the `import/resolve` extension for `.vue`.
